lang/compiler_test.go

```diff
@@ -474,7 +474,7 @@ func (rule removeCount) Name() string {
 func (rule removeCount) Pattern() plan.Pattern {
 	return plan.Pat(universe.CountKind, plan.Any())
 }
-func (rule removeCount) Rewrite(ctx context.Context, node plan.Node) (plan.Node, bool, error) {
+func (rule removeCount) Rewrite(ctx context.Context, node plan.Node, nextNodeId *int) (plan.Node, bool, error) {
```

Comment:
This pattern, where we add a new parameter without using it, often indicates to me that we've got a leaky interface or abstraction. I see this pattern _a lot_ in this patch, so wondering you have thoughts about it.